config/ios/SystemProjectConfig.h

```diff
@@ -0,0 +1,26 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      chip::System project configuration for standalone builds on iOS
+ *
+ */
+#ifndef SYSTEMPROJECTCONFIG_H
```

Comment:
why not `#pragma once` ?

Reply:
Sorry copied from config/standalone/SystemProjectConfig.h which apparently wasn't migrated.